packages/ember-data/lib/system/debug/debug-adapter.js

```diff
@@ -42,11 +43,15 @@ export default Ember.DataAdapter.extend({
     return columns;
   },
 
-  getRecords: function(modelNameOrFactory) {
-    // TODO: Ask Teddy what we should do here.
-    // Ideally this should always get passed a string.
-
-    var modelName = typeof modelNameOrFactory === 'string' ? modelNameOrFactory : modelNameOrFactory.modelName;
+  getRecords: function(klass, modelName) {
+    if (arguments.length < 2) {
+      // Legacy Ember.js < 1.13 support
+      let containerKey = klass._debugContainerKey;
+      if (containerKey) {
+        modelName = containerKey.match(/model:(.*)/)[1];
```

Comment:
Yes, but I couldn't think of a reason why it wouldn't match. Do you think there's a case where the model factory's container key doesn't contain `model:`?